lib/vfscore/automount.c

```diff
@@ -687,7 +687,10 @@ static inline int vfscore_mount_volume(const struct vfscore_volume *vv)
 		    vv->ukopts);
 
 	if (vv->ukopts & LIBVFSCORE_UKOPT_MKMP) {
-		rc = vfscore_ukopt_mkmp(path);
+		char tmp[strlen(path) + 1];
+
+		strcpy(tmp, path);
+		rc = vfscore_ukopt_mkmp(tmp);
```

Comment:
Looks like `vfscore_ukopt_mkmp()` modifies the temp path, while later we pass `path` to `mount()`. Is it safe to do so?

Reply:
Thanks for confirming